conjure-java-core/src/integrationInput/java/com/palantir/product/Union.java

```diff
@@ -207,7 +208,7 @@ public T visitUnknown(String value) {
         private final String value;
 
         @JsonCreator
-        private FooWrapper(@JsonProperty("foo") @Nonnull String value) {
+        private FooWrapper(@JsonSetter("foo") @Nonnull String value) {
```

Comment:
what effect does the JsonProperty -> JsonSetter change have?

Reply:
Yep, they split the JsonProperty annotation into JsonGetter and JsonSetter for specificity and additional feature support.